util/printchplenv

```diff
@@ -156,8 +159,7 @@ def print_mode(mode='list', anonymize=False):
         print_var('  CHPL_JEMALLOC_UNIQ_CFG_PATH',
                   chpl_3p_jemalloc_configs.get_uniq_cfg_path(),
                   mode, filters=('runtime',))
-        if target_mem == 'jemalloc':
-            link_args_3p.extend(chpl_3p_jemalloc_configs.get_link_args())
+        link_args_3p.extend(chpl_3p_jemalloc_configs.get_link_args(jemalloc))
```

Comment:
Philosophically, I'd prefer to have chpl_3p_jemalloc_configs.get_link_args be totally responsible for any use of jemalloc. However everything would work OK with the target_mem condition. I just viewed adding CHPL_JEMALLOC as separating out these portions. E.g. if we ever had a use of jemalloc that was not dependent on CHPL_MEM.